src/client_side_request.cc

```diff
@@ -1927,7 +1927,7 @@ ClientHttpRequest::handleAdaptedHeader(Http::Message *msg)
         setLogUri(this, urlCanonicalClean(request));
         assert(request->method.id());
     } else if (HttpReply *new_rep = dynamic_cast<HttpReply*>(msg)) {
-        debugs(85,3,HERE << "REQMOD reply is HTTP reply");
+        debugs(85,3,"REQMOD reply is HTTP reply");
```

Comment:
This PR automatically removes HERE, and nothing else should be required. The spacing issue is covered in another comment, so I will not repeat those arguments here.